apiserver/facades/client/spaces/spaces.go

```diff
@@ -461,3 +555,43 @@ func (api *API) getApplicationsBindSpace(givenSpaceID string) ([]string, error)
 	}
 	return applications.SortedValues(), nil
 }
+
+// getConstraintsChanges will do nothing if there are no spaces constraints to update
```

Comment:
// getConstraintsChanges gets the current constraints to update.
// will do nothing if there are no spaces constraints to update